syntax = "proto3";

import "common.proto";
import "lane_multitask.proto";
import "perception_base.proto";
package autodrive.workflow.proto;

message SingleCamLane {
  int32 camera_id = 1;
  repeated LaneParsingImageData lane_parsing_image = 2;
  perception_base.Rect parsing_roi = 3;
}

message LaneOnlCalibMsg {
  // Header
  MsgHeader header = 1;
  // Error
  Error error = 2;
  repeated SingleCamLane camera_lane = 3;
}